packages/broker/test/integration/plugins/metrics/node/NodeMetrics.test.ts

```diff
@@ -20,27 +20,43 @@ describe('NodeMetrics', () => {
     let streamIdPrefix: string
 
     beforeAll(async () => {
-        const tmpAccount = new Wallet(await fetchPrivateKeyWithGas())
+        const brokerWallet = new Wallet(await fetchPrivateKeyWithGas())
 
-        nodeAddress = tmpAccount.address
+        nodeAddress = brokerWallet.address
         tracker = await startTestTracker(trackerPort)
-        client = await createClient(tracker, tmpAccount.privateKey)
+        client = await createClient(tracker, brokerWallet.privateKey)
 
         const stream = await client.createStream({
             id: `/metrics/nodes/${uuid()}/sec`,
             partitions: NUM_OF_PARTITIONS
         })
         await stream.grantPermissions({ permissions: [StreamPermission.SUBSCRIBE], user: nodeAddress })
         streamIdPrefix = stream.id.replace('sec', '')
+        // a previous test run may have created the assignment stream
+        await client.getOrCreateStream({
+            id: '/assignments'
+        })
```

Comment:
Should this be a public stream, or does it matter in this case?

Reply:
No need for that in this tests